modules/library/render/src/test/java/org/geotools/renderer/lite/FootprintsRenderingTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2021, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.renderer.lite;
+
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+import org.apache.commons.io.IOUtils;
+import org.geotools.gce.imagemosaic.ImageMosaicReader;
+import org.geotools.gce.imagemosaic.ImageMosaicReaderTest;
+import org.geotools.geometry.jts.ReferencedEnvelope;
+import org.geotools.image.test.ImageAssert;
+import org.geotools.map.GridReaderLayer;
+import org.geotools.map.MapContent;
+import org.geotools.styling.Style;
+import org.geotools.test.TestData;
+import org.junit.Test;
+
+public class FootprintsRenderingTest {
+
+    @Test
+    public void testFootprintsTransformation() throws Exception {
+        File zip = new File("./target/rgb.jar");
```

Comment:
I have no clue... just used a test mosaic that was already there, in the gt-imagemosaic plugin.
It has been there longer than the github history allows to investigate, not sure how or why.
Used it because it's easy to pick from the classpath a single file, while it's harder to get into a "folder" and copy all of its contents.